snet_cli/mpe_service_command.py

```diff
@@ -0,0 +1,101 @@
+from snet_cli.commands    import BlockchainCommand
+import snet_cli.utils_ipfs as utils_ipfs
+from snet_cli.mpe_service_metadata import mpe_service_metadata, load_mpe_service_metadata
+from snet_cli.utils import type_converter
+import base58
+
+class MPEServiceCommand(BlockchainCommand):
+        
+    # I. Low level functions
+    
+    # publis proto files in ipfs and print hash
+    def publish_proto_in_ipfs(self):
+        ipfs_hash_base58 = utils_ipfs.publish_proto_in_ipfs(self._get_ipfs_client(), self.args.protodir)
+        self._printout(ipfs_hash_base58)
+        
+    # Init metadata and set model_ipfs_hash  
+    def metadata_init(self):
+        metadata = mpe_service_metadata()
+        metadata.set_model_ipfs_hash(self.args.model_ipfs_hash)
+        metadata.set_mpe_address(self.args.mpe_address)
```

Comment:
Side note on keeping MultiPartyEscrow (MPE) contract address in metadata.

It is required to correctly handle MPE contracts updates in both client and service sides. MPE contract address is used in signed payment message to guarantee that both payment sender and payment receiver use the same data source to check the payment. 

When data is kept in contract then contract address changes during contract upgrade and service provider should publish new Service with new metadata to incorporate change.

When data and logic of contract are decoupled the upgrade may be seamless for sender and receiver as  most of upgrades doesn't affect data contract address. And changing this address in metadata is required only when data part of the contract is updated.